operator/cmd/mesh/install.go

```diff
@@ -190,38 +189,14 @@ func Install(rootArgs *RootArgs, iArgs *InstallArgs, logOpts *log.Options, stdOu
 		return fmt.Errorf("could not configure logs: %s", err)
 	}
 
-	// Detect whether previous installation exists prior to performing the installation.
-	exists := revtag.PreviousInstallExists(context.Background(), kubeClient.Kube())
-	rev := iop.Spec.Revision
-	isDefaultInstallation := rev == "" && iop.Spec.Components.Pilot != nil && iop.Spec.Components.Pilot.Enabled.Value
-	operatorManageWebhooks := operatorManageWebhooks(iop)
-
 	iop, err = InstallManifests(iop, iArgs.Force, rootArgs.DryRun, kubeClient, client, iArgs.ReadinessTimeout, l)
 	if err != nil {
 		return fmt.Errorf("failed to install manifests: %v", err)
 	}
-
-	if !operatorManageWebhooks && (!exists || isDefaultInstallation) {
+	if processed, err := istiocontrolplane.ProcessDefaultWebhook(kubeClient, iop, ns); err != nil {
+		return fmt.Errorf("failed to process default webhook: %v", err)
+	} else if processed {
 		p.Println("Making this installation the default for injection and validation.")
```

Comment:
nit: technically we already made it the default at this point, but we say we are "Making..."